scripts/arangodb-docker-container-setup/docker-container-scripts/test-data/testData.json

```diff
@@ -1,45 +1,45 @@
 {
     "tags": [
         {
+            "type": "tag",
             "_key": "0",
             "published": true,
-            "type": "tag",
             "allowedContextTypes": [
```

Comment:
Would this be saved in Arango?

Reply:
Good question. I don't think it's necessary. There are two paths I can think of:
1.) make getAllowedContextTypes a method instead
2.) remove this property when calling `toDTO`

I might know a clever hack- we could make `allowedContextTypes` a getter. 

I'll see if I can solve this before merging in.